source/Nuke.Common/Execution/Logging.cs

```diff
@@ -52,7 +52,9 @@ public static void Configure(NukeBuild build = null)
                 .ConfigureHost(build)
                 .ConfigureConsole(build)
                 .ConfigureInMemory(build)
-                .ConfigureFiles(build)
+                //todo: if the file is locked, then nuke silently fails.
```

Comment:
Fought with this for a while, then decided to ignore it for now.
Obviously, something we should do a better way